### ISSUE TITLE
Heroku deployment fails with strong-mesh-client

### ISSUE THREAD

**Issue by mikeodell77** [strongloop/strong-mesh-client#56]
When I deploy to heroku, the deploy fails with the following:

```202091 error Failed at the strong-mesh-client@1.6.4 install script 'node proxy/build-client.js'.```

I see that there was another issue #47 , but somehow it resolved itself.

**Comment by cleondz** [comment]
I'm facing the same issue on install strongloop in our servers

```
  npm ERR! argv "bin/node" "bin/npm" "install" "-g" "strongloop"
  npm ERR! node v4.4.6
  npm ERR! npm  v2.15.5
  npm ERR! code ELIFECYCLE
  
  npm ERR! strong-mesh-client@1.6.4 install: `node proxy/build-client.js`
  npm ERR! Exit status 1
```
I tried installing strongloop previous version but not luck , some workaround on this?

**Comment by rmg** [comment]
This should have been resolved with #58 and `strong-mesh-client@1.6.5`.

@mikeodell77 please re-open if you are still having this problem.
